pages/poi/[id]/details.vue

```diff
@@ -74,10 +75,20 @@ if (!poi.value) {
 // Computed
 //
 const pageTitle = computed(() => {
-  if (!poi.value)
+  if (!poi.value || !poi.value.properties.name)
     return ''
 
-  return `${poi.value.properties.classe} - ${poi.value.properties.name}`
+  const featureName = poi.value.properties.name
+
+  if (poi.value.properties.editorial) {
+    if (poi.value.properties.editorial.class_label_details)
+      return `${poi.value.properties.editorial.class_label_details[locale.value]} - ${featureName}`
+
+    if (poi.value.properties.editorial.class_label)
+      return `${poi.value.properties.editorial.class_label[locale.value]} - ${featureName}`
+  }
```

Comment:
class_label_details should be defined, but without value for the locale.

Reply:
We have to update our types then, because for the moment we have the following:
```
'editorial'?: {
    'popup_fields'?: FieldsListItem[]
    'details_fields'?: FieldsList
    'list_fields'?: FieldsListItem[]
    'class_label'?: MultilingualString
    'class_label_popup'?: MultilingualString
    'class_label_details'?: MultilingualString
    'website:details'?: string
    'unavoidable'?: boolean
  }
```

But maybe we should have a global rework of our API types to ensure we are in sync with new Elasa standards ?